src/output/icons.rs

```diff
@@ -113,12 +122,19 @@ impl Icons {
 /// Mapping from full filenames to directory icon. This mapping should contain
 /// all the directories that have a custom icon.
 const DIRECTORY_ICONS: Map<&'static str, char> = phf_map! {
+    "__pycache__"         => Icons::LANG_PYTHON,    // 󰌠
```

Comment:
We have to decide if we want non-folder icons for __pycache__, .cargo, .doom.d, and .emacs.d

Reply:
I understand why others might not want this, because they are out of place in the directory icons. I can change this.

Reply:
I think icons for those folder is good

Reply:
We have decided we do not want icons for these unless they are folder icons.

Reply:
Sounds good